languages/ruby/exercises/concept/booleans/.docs/introduction.md

```diff
@@ -0,0 +1,61 @@
+## True, False, and Nil
+
+True and false logical states are represented with `true` and `false` in Ruby. These may either be used or returned as literals on their own, or as a result of logical or comparison methods.
+
+```ruby
+true_variable = true
+false_variable = false
+
+true && false
+# => false is returned as a result of the logical method
+
+1 < 2
+# => true is returned as a result of the comparison method
+```
+
+[Nil][nil-dictionary] is an English word meaning "nothing" or "zero". In Ruby, `nil` is a value used to express the _absence_ of an object.
+
+```ruby
+# I do not have a favorite color
+favorite_color = nil
+```
+
+In other programming languages, `null` or `none` values might play a similar role.
+
+## _Truthy_ and _falsey_
+
+In Ruby, all objects evaluate to a _truthy_ or _falsey_ value when they are encountered in a boolean context (like an `if` expression). All data is considered _truthy_ **except** for `false` and `nil`.
```

Comment:
```suggestion
In Ruby, all objects evaluate to a _truthy_ or _falsey_ value when they are encountered in a boolean context (conditional statement). All values and objects evaluated as _true_ and considered _truthy_ **except** for `false` and `nil`, referred to as _falsey_.
```

I still don't like my changes all that much, except for the conditional statement.

If statements should not produce a true or false value, if they do, it is often an indication that it should be a conditional statement instead.  The "truthy/falsey" comes from the if statement only because everything that is an object has this attribute, and the if statement will always return something when evaluated/invoked.

Reply:
```suggestion
In Ruby, all objects evaluate to a _truthy_ or _falsey_ value when they are encountered in a boolean context (conditional statement). All values and objects are considered _truthy_. Only `false` and `nil` are considered _falsey_ values.
```

maybe?

Reply:
All objects evaluate as _truthy_ or _falsey_. Not as a value, as the idea is there, but not the "truthy" value, _per se_. I would drop the word "value" at the end of the last sentence as well.

Reply:
Have we introduce the difference in definition of "boolean" and "Boolean" as we use it?  The "boolean" to indicate that we aren't necessarily looking for the formal `true` or `false` objects, but the _truthy_ or _falsey_ evaluation?

Reply:
would `logical context` be more clear?

Reply:
As opposed to "conditional statement" or "boolean" or…